task_15/app.js

```diff
@@ -0,0 +1,37 @@
+const express = require('express')
+const app = express()
+const port = 3000
+const {remote} = require('webdriverio')
+
+app.use(express.json())
+
+app.get('/', (req, res)=>{
+
+    ;(async () => {
+        const browser = await remote({
+            capabilities: {
+                browserName: 'chrome'
+            }
+        })
+
+        await browser.url('https://webdriver.io')
+
+        const apiLink = await browser.$('=API')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/jkalandarov/epam_stage_2/pullRequest?prid=8887246)
```suggestion
        const apiLink = await browser.$("=API")
```